tsdb/db.go

```diff
@@ -569,11 +569,11 @@ func (db *DBReadOnly) Close() error {
 }
 
 // Open returns a new DB in the given directory. If options are empty, DefaultOptions will be used.
-func Open(dir string, l log.Logger, r prometheus.Registerer, opts *Options, stats *DBStats) (db *DB, err error) {
+func Open(ctx context.Context, dir string, l log.Logger, r prometheus.Registerer, opts *Options, stats *DBStats) (db *DB, err error) {
```

Comment:
We need a comment for this function mentioning that the context is only used to abort the WAL replay.

Reply:
Added a comment